pass-an-image.cpp

```diff
@@ -1,12 +1,31 @@
 #include <iostream>
+#include <memory>
+#include <random>
 #include "brightener.h"
 
 int main() {
-    Image image;
-    image.rows = 512;
-    image.columns = 512;
-	std::cout << "Brightening a 512 x 512 image\n";
-    ImageBrightener brightener(image);
+    // Use std::unique_ptr for automatic memory management
+    auto image = std::make_unique<Image>();
+    image->rows = 512;
+    image->columns = 512;
+
+    // Initialize random number generator
+    std::random_device rd;
```

Comment:
Nice initialization for variety of input. I guess this will yield different `attenuatedCount` every time you run it?
Can you think of an easy way to retain some variety, and still be able to say if the code is correct?